File: su/src/main/java/org/sqlbuilder/su/SuSynsetResolver.java
package org.sqlbuilder.su;

import org.sqlbuilder.common.Resolver;
import org.sqlbuilder2.ser.DeSerialize;

import java.io.File;
import java.io.IOException;

public class SuSynsetResolver extends Resolver<String, Integer>
{
	public SuSynsetResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: su/src/main/java/org/sqlbuilder/su/SuSynsetResolver.kt
package org.sqlbuilder.su

import org.sqlbuilder.common.DeSerialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias SuSynsetResolvable = String
typealias SuSynsetResolved = Int

class SuSynsetResolver(ser: String) : Resolver<SuSynsetResolvable, SuSynsetResolved>(DeSerialize.deserialize<Map<SuSynsetResolvable, SuSynsetResolved>>(File(ser)))
